src/main/resources/liquibase/crop_changelog/24_5_0.xml

```diff
@@ -104,4 +104,598 @@
+
+        </sql>
+    </changeSet>	
+	
+    <changeSet id="v24.5.0-15" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Plant Number" AND cv_id = 1010;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add property Plant Number </comment>
+        <sql dbms="mysql" splitStatements="true">
+			set @class = ( SELECT cvterm_id FROM cvterm WHERE name = "Passport" AND cv_id = 1011);
+			
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1010, 'Plant Number', 'Plant Number', null, 0, 0);
+									
+			set @propertyId = (select cvterm_id from cvterm where name = 'Plant Number' AND cv_id = 1010);
+			
+			INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1225, @propertyId,@class );
+			
+        </sql>
+    </changeSet>				
+    
+	<changeSet id="v24.5.0-16" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "PLANT_NUMBER_AP_text" and cv_id = 1040 ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add variable PLANT_NUMBER_AP_text</comment>
+        <sql dbms="mysql" splitStatements="true">
+            set @methodID = (select cvterm_id from cvterm where name = 'Assigned Passport Descriptor' AND cv_id = 1020);
+            set @scaleId = (select cvterm_id from cvterm where name = 'Text' AND cv_id = 1030);
+			set @propertyId = (select cvterm_id from cvterm where name = 'Plant Number' AND cv_id = 1010);
+			
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1040, 'PLANT_NUMBER_AP_text', 'Plant Number', null, 0, 0);
+
+            set @variableId = (select cvterm_id from cvterm where name = 'PLANT_NUMBER_AP_text' AND cv_id = 1040);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1200, @variableId, @propertyId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1210, @variableId, @methodId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1220, @variableId, @scaleId);
+
+            INSERT INTO cvtermprop(cvterm_id, type_id, value, rank)
+            VALUES (@variableId, 1800, 'Germplasm Passport', 0);
+			
+			INSERT INTO cvtermprop (cvterm_id, type_id, value, rank) 
+			VALUES (@variableId, 8048, (SELECT DATE_FORMAT(CURRENT_TIMESTAMP(), '%Y-%m-%dT%T.000Z')), 0);
+
+        </sql>
+    </changeSet>	
+	
+	  
+	<changeSet id="v24.5.1-1" author="rowena">
+        <preConditions onFail="MARK_RAN">
+			<and>
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(1) > 0 FROM listnms where listdate &lt; 20230000;
```

Comment:
@rvtulod This precondition should check for max(entryid) <> count(entryid) in listdata, instead of the listnms.listdate